airbyte-webapp/src/packages/cloud/views/auth/Auth.tsx

```diff
@@ -39,7 +59,11 @@ const Auth: React.FC = () => {
         </FormContent>
       </div>
       <div className={styles.rightSide}>
-        <PersonQuoteCover />
+        {hasValidRightSideUrl(rightSideUrl) ? (
+          <iframe src={rightSideUrl} title="Right Side" scrolling="no" />
```

Comment:
Since the title might be read out by assistive technology, I would suggest we change this to something that gives a bit more hint about the iframe's content, e.g. `Why you should use Airbyte` or `Benefits of using Airbyte`. Also I think we should ideally i18n this string.